<launch>

  <arg name="dir_name" default="$(env PWD)"/>
  <arg name="prefix" default="rec"/>
  <arg name="node_start_delay" default="1.0"/>

  <node launch-prefix="/bin/bash -c 'sleep $(arg node_start_delay); $0 $@' " pkg="fri_sawyer" type="recorder" name="$(eval anon('recorder'))" required="true" output="screen" cwd="node">
    <param name="dir_name" value="$(arg dir_name)" />
    <param name="prefix" value="$(arg prefix)" />
  </node>


</launch>
